deploy/cloudwash_job.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: provisioning-cleanup-resources
+objects:
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: provisioning-cleanup-${IMAGE_TAG}-${UID}
+    annotations:
+      "ignore-check.kube-linter.io/no-liveness-probe": "probes not required on Job pods"
+      "ignore-check.kube-linter.io/no-readiness-probe": "probes not required on Job pods"
+  spec:
+    backoffLimit: 0
+    template:
+      spec:
+        imagePullSecrets:
+        - name: quay-cloudservices-pull
+        restartPolicy: Never
+        containers:
+        - name: provisioning-cleanup-${IMAGE_TAG}-${UID}
```

Comment:
Why is the IMAGE_TAG in the name here? :thinking:

Reply:
So, am not sure either but it should be for the container's unique name.

I think it's like default things which you need to add for every job that will run inside the container /?

I haven't added it at first but then @tpapaioa corrected me there.

Reply:
Ok :shrug: I'm bit confused why it is an IMAGE_TAG as we do not use it as IMAGE_TAG, but there might be something I do not understand :shrug:

Reply:
I will let @tpapaioa answer this. :) 
Thanks!

Reply:
https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/continuous-testing-in-app-interface.md#job-definition-and-naming